### PR TITLE
feat: perf and a11y updates

### PR DESCRIPTION
**Pull request by aulneau**
#### Before
Screen Shot 2020-04-21 at 3 51 05 PM

#### After
Screen Shot 2020-04-21 at 3 50 25 PM

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -7,25 +11,42 @@ module.exports = {
...
   webpack(config, { dev, isServer }) {
     const splitChunks = config.optimization && config.optimization.splitChunks;
     if (splitChunks) {
       const cacheGroups = splitChunks.cacheGroups;
-      const reactModules = /[\\/]node_modules[\\/](react|react-dom)[\\/]/;
+      const preactModules = /[\\/]node_modules[\\/](preact|preact-render-to-string|preact-context-provider)[\\/]/;
       if (cacheGroups.framework) {
-        cacheGroups.react = Object.assign({}, cacheGroups.framework, {
-          test: reactModules,
+        cacheGroups.preact = Object.assign({}, cacheGroups.framework, {
```
Comment (kyranjamie) [inline next.config.js:21]: Is there a big gain in switching for preact?

File: server.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  send: ({ data, res }) => res.send(data),
+});
+
+app.prepare().then(() => {
+  const server = express();
+
+  server.get('/', (req, res) => ssrCache({ req, res, pagePath: '/' }));
+
+  server.get('/txid/:txid', (req, res) => {
```
Comment (kyranjamie) [inline server.js:33]: This renders once, and returns cache for the next hour? What about txs that are pending and have a status that'll soon change?

File: src/common/fonts.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import FontFaceObserver from 'fontfaceobserver';
+
+export const handleFontLoading = async () => {
+  const link = document.createElement('link');
+  link.href =
+    'https://fonts.googleapis.com/css2?family=Fira+Code&family=Inter:wght@400;500;600;700&display=swap';
+  link.rel = 'stylesheet';
+  document.head.appendChild(link);
+  const inter = new FontFaceObserver('Inter');
+  const fira = new FontFaceObserver('Fira Code');
```
Comment (kyranjamie) [inline src/common/fonts.ts:10]: ah, we're loading Fira..

File: server.js (hunk 2 above)
Comment (aulneau) [reply server.js:33]: Yeah valid question. We can shorten the time. It drastically improves perf to have a cached version. Seems like right now the perf hit comes from the time it takes to fetch on the server to the sidecar. I wonder what if any kind of caching the sidecar will have?

File: next.config.js (hunk 1 above)
Comment (aulneau) [reply next.config.js:21]: Screen Shot 2020-04-22 at 9 01 47 AM

Lighthouse with React:
Screen Shot 2020-04-22 at 8 58 24 AM

Lighthouse with Preact:
Screen Shot 2020-04-22 at 9 00 00 AM

File: server.js (hunk 2 above)
Comment (kyranjamie) [reply server.js:33]: I think it might be best to not use it for tx page, on the hypothesis the average number of page views per tx will be < 1.

**Pull request merged**